src/app/(BottomTabNavigation)/AllCases/QRCodeScanner/index.tsx

```diff
@@ -22,9 +29,29 @@ function QRCodeScannerScreen() {
   const [scanned, setScanned] = useState<boolean>(false);
   const [validIds, setValidIds] = useState<CaseUid[]>([]);
   const [userIds, setUserIds] = useState<CaseUid[]>([]);
-  const [toast, setToast] = useState<string>();
+  const [userCase, setUserCase] = useState<Case>();
   const navigation = useNavigation();
 
+  const toastConfig: ToastConfig = {
+    success: (props: any) => <BaseToast />,
+
+    error: (props: any) => (
+      <ErrorToast
+        {...props}
+        text1Style={{
+          fontSize: 17,
+        }}
+        renderLeadingIcon={ErrorIcon}
+        style={{
+          borderLeftColor: colors.darkRed,
```

Comment:
The toast on the Figma is slightly smaller and there is no red left border so let's update this to reflect that and once again place styling in the style sheet